cli/src/main/java/org/openjdk/skara/cli/GitWebrev.java

```diff
@@ -454,6 +458,10 @@ private static void generate(String[] args) throws IOException {
                   .comments(comments)
                   .generate(base, head);
         }
+        if (!quiet) {
+            System.out.println("Webrev executed successfully, details are in the link below:");
+            System.out.println("file://" + new File(output.resolve("index.html").toUri()).getAbsoluteFile());
```

Comment:
That looks like a weird way to generate a URL. Why not just:
```suggestion
            System.out.println(output.resolve("index.html").toUri());
```

Reply:
Yeah, will do the change